modules/core/04-channel/keeper/packet_test.go

```diff
@@ -20,7 +20,7 @@ import (
 var (
 	disabledTimeoutTimestamp = uint64(0)
 	disabledTimeoutHeight    = clienttypes.ZeroHeight()
-	timeoutHeight            = clienttypes.NewHeight(1, 100)
+	defaultTimeoutHeight     = clienttypes.NewHeight(1, 100)
```

Comment:
I had to change this back since it was conflicting with the local var in `TestSendPacket`. I also just find it makes more sense, since there are some non default values I put in the test cases.